ImageManager/ImageManager.py

```diff
@@ -59,12 +99,44 @@ def create_image():
         except IntegrityError as error:
             handle_consistency_exception(error)
 
+        else:
+            result = {'message': 'image created, awaiting upload',
+                      'url': url_for('image.upload_image', imageid=imageid)}
+
+        Timer(UPLOAD_TIMEOUT, confirm_image_uploaded, [imageid]).start()
```

Comment:
If I'm reading this right, this would "tie" the upload verification procedure to this instance (process) of the image manager.

Perhaps storing a timestamp somewhere and using that as a timeref when validating the image upload would be a safer approach.